app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(command: str) -> None:
+    list_ = command.split()
+    first_name = str(list_[1])
+    second_name = str(list_[2])
```

Comment:
The variable names `first_name` and `second_name` are not clear in the context of this function. It would be better to use names that reflect their purpose in the function, like `source_file` and `destination_file`.